code/modules/buildmode/submodes/advanced.dm

```diff
@@ -32,7 +33,15 @@
 	var/list/pa = params2list(params)
 	var/left_click = pa.Find("left")
 	var/right_click = pa.Find("right")
-	if(left_click)
+	var/alt_click = pa.Find("alt")
+
+	if(left_click && alt_click)
+		if ((istype(object, /turf) || istype(object, /obj) || istype(object, /mob))
```

Comment:
Drop the needless ()'s and it's good to go.

Reply:
imagine a future where i had the mental abilities to make a 2-line change witout a billion reviews im gonna die